codebase-pipelines/eventbridge.tf

```diff
@@ -29,7 +29,7 @@ resource "aws_iam_role" "event_bridge_pipeline_trigger" {
 }
 
 resource "aws_iam_role_policy" "event_bridge_pipeline_trigger" {
-  name   = "${var.application}-${var.codebase}-pipeline-trigger-access-for-event-bridge"
+  name   = "event-bridge-access"
```

Comment:
There will only ever be one of these?

Reply:
It doesn't need to be unique because it's an inline role policy so only exists on that role